### PR TITLE
DOC-3744: Update course updates and handouts

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3744](https://openedx.atlassian.net/browse/DOC-3744)

Update Course Handouts and Updates chapter of the B&R guide to reflect that we are encouraging only a single course update = welcome message that can be dismissed by learners from the **Course** page.

This PR might not address the complete/future implementation of unified course messaging per LEARNER-154 but should be published as soon as possible to bring the course author documentation as close as possible to what is currently live on edx.org.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @robrap 
- [x] Subject matter expert: @HarryRein 
- [x] Doc team review: @edx/doc
- [x] Product review: @marcotuts 

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [x] Squash commits

### CODE REVIEW

File: en_us/shared/course_assets/handouts_updates.rst [hunk 1]
```diff
@@ -22,18 +39,25 @@ course updates and handouts on the **Home** page in your course.
 Add a Course Update
 **********************
 
-You add updates to notify learners of upcoming exams or deadlines, changes in
-the course schedule, or to make other announcements. Updates are visible to
-enrolled learners as soon as you post them.
+You create course updates in Studio. When you publish an update, learners who
+are entering the course for the first time see the update on the **Course**
```
Comment (robrap) [inline en_us/shared/course_assets/handouts_updates.rst]: "see the latest update"?

File: en_us/shared/course_assets/handouts_updates.rst [hunk 2]
```diff
@@ -22,18 +39,25 @@ course updates and handouts on the **Home** page in your course.
...
-does not control when the date becomes visible to learners. In the LMS, updates
-display on the **Home** page in reverse chronological order (most recent at the
-top), based on the date associated with each update.
+.. note:: Learners can dismiss the update that displays on their **Course**
+   page after they have read it. When learners have dismissed a message, they
+   no longer see new updates on the **Course** page, but can access the
+   dismissed message and any other updates from the **Updates** link in the
+   sidebar on the **Course** page.
+
+Each update requires a date only for identification and sorting purposes. This
```
Comment (robrap) [inline en_us/shared/course_assets/handouts_updates.rst]: I think date does control the Welcome Message for new learners.  It will be the latest message.

**Review by robrap — COMMENTED** [review]
Minor comments.  I'm not the right person to review handouts. :)

**Review by grantgoodmanedX — APPROVED** [review]
Looks good 👍

File: en_us/shared/course_assets/handouts_updates.rst (hunk 2 above)
Comment (catong) [reply en_us/shared/course_assets/handouts_updates.rst]: @robrap What this text is trying to say is that the date is not the publishing date. It's only used for sorting purposes on the **Updates** page in the LMS. That's how it worked and still works, I think. Regardless of the date you enter, each new update you create becomes "the latest update".

Comment (robrap) [reply en_us/shared/course_assets/handouts_updates.rst]: I haven't used it.  I thought @marcotuts said someone can set a distant future date to make it the latest and make it appear as the Welcome Message.  In any case, if @marcotuts could review this that would be best.

Comment (catong) [reply en_us/shared/course_assets/handouts_updates.rst]: Whatever update you create last becomes the latest, as far as I can tell by testing whatever is on Prod right now, regardless of the date you enter. @marcotuts can you please confirm whether this functionality has been changed?

Comment (marcotuts) [reply en_us/shared/course_assets/handouts_updates.rst]: @catong - that would be news to me, as we've messaged this out as being something that was only shown for 7 days and what we show is the most recent, not the most recently authored. I can test later tonight to understand what is going on, and put into a ticket for next sprint the gap between current behavior and what we're expecting and have messaged.

Comment (catong) [reply en_us/shared/course_assets/handouts_updates.rst]: Update: confirmed with @marcotuts that learners do see a single update - the one that has the LATEST DATE. Course teams in Studio see the list of course updates in order of creation, with the most recently created at the top. Also, "7 days expiry" is not implemented.

**Comment by catong** [comment]
@robrap @HarryRein @marcotuts @grantgoodmanedX - I've updated the PR to reflect confirmed behavior (re dates). Can you please do a final review and give thumbs if ready to go? Thanks.

**Review by marcotuts — APPROVED** [review]

**Review by grantgoodmanedX — APPROVED** [review]
Still looks good, with a comment on the first sentence that I made orally.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1559

**Pull request merged**